crates/obelisk/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+use std::os::raw::c_char;
+
+extern "C" {
+    pub fn run_node(code: *const c_char) -> i32;
```

Comment:
Is there gonna be a version of this that works with file paths instead, rather than a string of code?

Reply:
Yes! There should at least be a way to run a file! The official examples only have an analogy of `node -e` - that is something to work on.